### ISSUE TITLE
Problems with attachments on Calendar items

### ISSUE THREAD

**Issue by fooeynet** [ExchangeCalendar/exchangecalendar#13]
I cannot seem to retrieve attachments added to Calendar items.  Usually there is a long delay opening these items (see timestamps from log below), and then the attachment URLs show up as 
'somewhere?id=blahblahblah'

I'm using the 4.0.0-beta1 build.  This worked before I had to upgrade to these forked builds for later Thunderbird/Lightning support.  Does this work for others?
```
1st-setup:2017-5-3 11:17:48.663:[OM] getInvitedAttendee 2:mailto:XXXXXXXXXXXXXX (_getInvitedAttendee in mivExchangeCalendar.js:3063)
1st-setup:2017-5-3 11:17:48.664:[OM] getInvitedAttendee FOUND someonelse:Interview scheduled for XXXXXXX, attendee.id:mailto:XXXXXXXX
.com, attendee.participationStatus:NEEDS-ACTION (_getInvitedAttendee in mivExchangeCalendar.js:3071)
1st-setup:2017-5-3 11:17:48.664:[OM] getInvitedAttendee 2:mailto:XXXXXXXXXXXX (_getInvitedAttendee in mivExchangeCalendar.js:3063)
1st-setup:2017-5-3 11:17:48.664:[OM] getInvitedAttendee FOUND myself:Interview scheduled for XXXXXXXXXXXX, attendee.participationStatus:ACCEPTED, a
Item.myResponseType:Accept (_getInvitedAttendee in mivExchangeCalendar.js:3066)
1st-setup:2017-5-3 11:18:43.708:  -- It is an Exchange Calendar event:Interview scheduled for XXXXXXX
1st-setup:2017-5-3 11:18:43.712:  -- Could not add exchange attachment buttons:TypeError: this._document.getElementById(...) is null
1st-setup:2017-5-3 11:18:43.713:exchWebService.attachments.addAttachmentsFromItem: foutje:TypeError: documentLink is null
1st-setup:2017-5-3 11:18:43.713:exchWebService.attachments.addAttachmentsFromItem: title:Interview scheduled for XXXXXXXXXXX
1st-setup:2017-5-3 11:18:43.714:  -- We have attachments:1 (_onLoad in attachments-view.js:267)
1st-setup:2017-5-3 11:18:43.714:  -- processing attachment: 0
1st-setup:2017-5-3 11:18:43.715:exchWebService.attachments.addAttachment

```

**Comment by fooeynet** [comment]
I should also note that the attachment is perfectly viewable from the original meeting request *email*, it's only when trying to open the attachment from the resulting calendar item that I'm seeing errors.

**Comment by advancingu** [comment]
Should be fixed in [v4.0.0-beta2](https://github.com/ExchangeCalendar/exchangecalendar/releases/tag/v4.0.0-beta2).
